app/furniture/marketplace/management_component.html.erb

```diff
@@ -6,18 +6,19 @@
     <%= marketplace.room.name %>
   </p>
 </div>
-<%= render onboarding_component %>
-<%= render CardComponent.new(classes: "mt-3") do |card| %>
-  <%= content %>
-
-  <% card.with_footer do %>
-    <nav class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-3">
-      <%= render button({child: :stripe_account}, icon: :money) if policy(marketplace).edit? %>
-      <%= render button({child: :products}, icon: :tag)  if policy(marketplace.products).index? %>
-      <%= render button({child: :delivery_areas}, icon: :map)  if policy(marketplace.delivery_areas).index? %>
-      <%= render button({child: :tax_rates}, icon: :receipt_percent)  if policy(marketplace.tax_rates).index? %>
-      <%= render button({child: :orders}, icon: :cart)  if policy(marketplace.orders).index? %>
-      <%= render button({child: :notification_methods}, icon: :bell)  if policy(marketplace.notification_methods).index? %>
-    </nav>
+<div class="flex flex-col space-y-3">
```

Comment:
This makes me think of the Evil Martian's "wrapped component" helper for ViewComponents, from their `view-component_contrib` gem. It basically lets you do exactly this, but as a tidier helper that you we could add to our `ApplicationComponent`:
https://github.com/palkan/view_component-contrib#wrapped-components

(This gem has some other goodies that we might benefit from, and they are actively adding new features to it as they work more with ViewComponent. This feels like maybe a good excuse to add it to our codebase, but in that case it definitely feels like a separate PR.)

Reply:
Interesting, I'll check it out. Seems like this could be useful. Although the `render?` api is somewhat of a reversal from the way I've built frontend. Generally I've worked from the perspective that higher level components define conditions for child rendering.